p2p/src/main/kotlin/maru/p2p/MaruIncomingRpcRequestHandler.kt

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright 2025 Consensys Software Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package maru.p2p
+
+import io.netty.buffer.ByteBuf
+import io.netty.buffer.ByteBufUtil
+import org.apache.tuweni.bytes.Bytes
+import tech.pegasys.teku.networking.p2p.peer.NodeId
+import tech.pegasys.teku.networking.p2p.rpc.RpcRequestHandler
+import tech.pegasys.teku.networking.p2p.rpc.RpcStream
+
+class MaruIncomingRpcRequestHandler : RpcRequestHandler {
```

Comment:
I think this class belongs to test sourceset

Reply:
This is one of the classes that we will have to implement to make everything work.